src/components/IconButton.js

```diff
@@ -0,0 +1,41 @@
+import styled, { css } from 'styled-components';
+import ScreenReaderOnly from './ScreenReaderOnly';
+
+export default function IconButton({ buttonText, children, ...buttonProps }) {
+  return (
+    <ButtonStyled {...buttonProps}>
+      {children}
```

Comment:
If I remember right, Jerry told us that we're not using {children} that way. I think it's only for storybook.

Reply:
Hey. this is alright. 
We talked about not using it like this `children={something}`,  instead use `<something>{children}</something>`